collector/mdadm_linux.go

```diff
@@ -39,6 +39,8 @@ type mdStatus struct {
 	mdName       string
 	isActive     bool
```

Comment:
`isActive` doesn't follow the go naming best practices. Please rename to `active` while you're on it.